stracker/items/models.py

```diff
@@ -0,0 +1,76 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from core.models import BaseAbstractModel
+from customers.models import Customer
+
+
+class Category(BaseAbstractModel):
+    """
+    Model to hold item categories (types)
+    """
+
+    name = models.CharField(max_length=255, verbose_name=_("Category Name"))
+
+    class Meta:
+        verbose_name = _("category")
+        verbose_name_plural = _("categories")
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Fridge(BaseAbstractModel):
+    """
+    Model to hold storage locations
+    """
+    name = models.CharField(verbose_name=_("name"), max_length=128, unique=True)
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Shelf(BaseAbstractModel):
+    """
+    Model to hold storage divisions
+    """
+    name = models.CharField(verbose_name=_("name"), max_length=128, unique=True)
+    fridge = models.ForeignKey(Fridge, verbose_name=_("Fridge"), on_delete=models.CASCADE)
+    owner = models.ForeignKey(Customer,verbose_name=_("Owner"), on_delete=models.CASCADE)
+
+    def __str__(self):
+        return f"{self.name} - {self.fridge}"
+
+
+class Item(BaseAbstractModel):
+    """
+    Model to hold item details
+    """
+    sku = models.CharField(verbose_name=_("SKU"), max_length=128, unique=True)
+    name = models.CharField(verbose_name=_("name"), max_length=128)
+    owner = models.ForeignKey(Customer, verbose_name=_("Owner"), on_delete=models.PROTECT)
+    shelf = models.ForeignKey(Shelf, verbose_name=_("Shelf"), on_delete=models.CASCADE)
+    categories = models.ManyToManyField(Category, verbose_name=_("Categories"))
+    description = models.TextField(max_length=2000, verbose_name=_("Description"))
+    qrcode = models.ImageField(verbose_name=_("QR Code"))
+    image = models.ImageField(verbose_name=_("Image"))
```

Comment:
ImageField alanı verirken upload_to parametresi verilmesi gerekmez mi?

Reply:
İyi olur ancak verilmezse de dosyaları settings.py'de belirtilen media adresine yüklüyor. 

Konuşmak istediğim konulardan biri bu. Yüklenen imajlar serverda mı tutulmalı, yoksa bir cms ajanı mı kullanılmalı?

Reply:
```suggestion
    image = models.CharField(verbose_name=_("Image"))
```
fotoğrafları db'de tutmamak daha sağlıklı olacaktır. hem de bizim için farklı sistemlerin entegrasyonalrı konusunda deneyim olacaktır.

Reply:
Katılıyorum, ne var ki AWS (ya da hangi sistemi kullanacaksak) entegrasyonu için çok da acele etmeye gerek yok. Şimdilik bu şekilde kalması, projenin tamamlanmasına yakın cds'e bağlanması taraftarıyım.